### DISCUSSION TITLE
Multiple colonies question

### DISCUSSION THREAD

**Discussion by pietru2004** [ldtteam/minecolonies#10032] [Q&A] [answered]
Could there be option to be Owner of multiple colonies without abandoning them? Mainly having in mind how many amazing styles of colonies there are. it is hard at times to manage what building was what style...
Also can I ask what ability to easily abandon their colonies mean in the config? (config edit ui by create)
![obraz](https://github.com/user-attachments/assets/cd326432-9cb8-4740-8629-ea1fa3058a82)

**Comment by Thodor12** [comment, marked as answer]
This is likely going to be an option in the future, when, we don't know. But countries (full description TBA) are planned.

**Reply by pietru2004** [reply]
hm... good to know... would you be able to say what allow infinite colonies does? I mean about making it easy to abandon colony

**Reply by Thodor12** [reply]
You will get an option in the colony creation screen for the next colony that will allow you to abandon the old one so you are able to create a new one.
Previously you had to so only through commands.

Note that abandoning does not mean deletion, the colony will become ownerless, however you can add yourself back as an officer to that colony, still allowing you to modify it afterwards.

**Reply by pietru2004** [reply]
ok thank you for your help
